pkg/workspace_types.go

```diff
@@ -34,7 +34,7 @@ type Workspace struct {
 	ID                       uint64
 	Namespace                string
 	UID                      string `valid:"stringlength(3|30)~UID should be between 3 to 30 characters,dns,required"`
-	Name                     string `valid:"stringlength(3|30)~Name should be between 3 to 30 characters,matches(^[a-z]([-a-z0-9]*[a-z0-9])?)~Name must start with an alphabetic character and otherwise contain alphanumeric characters or -,required"`
+	Name                     string `valid:"stringlength(3|30)~Name should be between 3 to 30 characters,matches(^[a-zA-Z]([-a-z0-9 ]*[a-z0-9])?)~Name must start with an alphabetic character and otherwise contain alphanumeric characters or -,required"`
```

Comment:
Is this going to accept spaces? These are valid names:

"CVAT"
"My CVAT Workspace"
"CVAT Workspace 1"

Reply:
@rushtehrani Yep. It's hard to see, but there is a space in the regex. 

I cleaned up the tests so its more obvious and added those as test cases.